backend/src/main/java/atlantbh/restaurants/services/CousineService.java

```diff
@@ -6,6 +6,11 @@
 import atlantbh.restaurants.repositories.CousineRepository;
 import org.springframework.stereotype.Service;
 
+import java.util.List;
+
 @Service
 public class CousineService extends BaseService<Cousine, CousineSortKeys, CousineFilterBuilder, CousineRepository> {
+    public List<Cousine> findAll() {
+        return repository.findAll();
```

Comment:
no

Reply:
fixed in 0cd4debab433260ab6a8267aa0b72b31db581edb
#sleepDeprived